openedx/core/djangoapps/video_config/models.py

```diff
@@ -131,3 +131,62 @@ def __unicode__(self):
             course_key=unicode(self.course_id),
             not_enabled=not_en
         )
+
+
+class VideoUploadsEnabledByDefault(ConfigurationModel):
+    """
+    Enables video uploads enabled By default feature across the platform.
+    """
+    # this field overrides course-specific settings
+    enabled_for_all_courses = BooleanField(default=False)
+
+    @classmethod
+    def feature_enabled(cls, course_id):
```

Comment:
I think we should use request cache here. What do you say?

Reply:
It will be used in course xmodule, and it does not have much multiple calls (unlike video component), as course listings are using `CourseSummary`.